tracing/test-log-support/Cargo.toml

```diff
@@ -8,4 +8,4 @@ edition = "2018"
 
 [dependencies]
 tracing = { path = "..", features = ["log", "log-always"] }
-log = { version = "0.4", features = ["std"] }
+log = { version = "0.4.0", features = ["std"] }
```

Comment:
what is the explicit `.0` doing here? i can't immediately think of a reason it would be needed...is this just a style change?

Reply:
Just a style change, yes, as this doesn't impact cargo's behavior

But it *does* change cargo-upgrade's behavior: cargo-upgrade by default won't increase the precision of dependency specification (and I couldn't find a flag to tell it to do so).

So in order to actually make sure the dependency specifications match what I had in the lockfile, I did a `s/"(\d+\.\d+)"/$1.0/g` to get cargo-upgrade to make the specifications match the lockfile.

I did try not doing this initially and it wasn't sufficient. Any remaining `.0` then is now an explicit note that we're observed to work with `.0`.